packages/shorebird_cli/lib/src/commands/run_command.dart

```diff
@@ -164,4 +166,23 @@ Future<void> _extractShorebirdEngine(
       extractArchiveToDisk(archive, targetPath);
     },
   );
+
+  const executables = [
+    'flutter/prebuilts/macos-x64/dart-sdk/bin/dart',
```

Comment:
Some of these paths will only work on Mac, no?

Reply:
yup the current shorebird engine only works on mac, android, arm64 but this will need to change once we support more platforms/archs